gapic-generator-cloud/cloud-rubocop.yml

```diff
@@ -0,0 +1,14 @@
+# This file is to be used as the config for formatting the generated cloud files.
+# We can't reference the generated .rubocop.yml file because Rubocop is so clevar
```

Comment:
clever?

Reply:
"Clevar" is even more clever than "clever".

Reply:
Rubocop is definitely a cleaver.